deploy/roles/jdbc/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+#Required for computer to make an ssl connection for postgres (since they sign their cert with letsencrypt)
+- name: Install cert utilities
+  yum: 
+    name: ca-certificates
+    state: latest
+  sudo: yes
```

Comment:
Nice, thanks!  FYI, we've generally been running everything as "sudo" in Aurora, so typically we have not specified sudo per task, but it's not a bad idea.  The newest version of Ansible does this slightly different though, so instead of:
```
sudo: yes
```
it should be:
```
become: yes
become_method: sudo
```